.github/workflows/config.yml

```diff
@@ -61,6 +61,7 @@ jobs:
       - name: Build documentation
         run: |
           cd docs
+          ruby source/data/create_default_schedules_csv_tables.rb
```

Comment:
I would prefer we remove the need for this script, it's just one more thing to maintain.

My suggestion: Use a single `csv-table` that points to default_schedules.csv, add it to the [Default Schedules](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#default-schedules) section, and then link to it from all the other schedule elements throughout the documentation.